src/components/Accounts.tsx

```diff
@@ -14,13 +14,19 @@
 // You should have received a copy of the GNU General Public License
```

Comment:
@goldsteinsveta : This component contains the account selector dropdown. Please take into account that component [Account](https://github.com/paritytech/parity-bridges-ui/pull/69/files#diff-3ad4937f6f0f4d3aedd698a6d1d6547b848be2e418fcd11b13b9131529183991) contains the custom style for the dropdown element.

Reply:
I'll probably rewrite it in pure MUI and remove styled-components dependency when will be styling this. Don't worry about it!